lib/internal/bootstrap_node.js

```diff
@@ -208,7 +207,6 @@
 
   function setupGlobalVariables() {
     global.process = process;
-    global.global = global;
```

Comment:
From https://github.com/nodejs/node/pull/5888#discussion_r57346386

Looks this is was duplicated and unnecessary.